bin/package_android

```diff
@@ -14,17 +14,21 @@ set -e
 source bin/helpers/output.sh
 source bin/helpers/functions.sh
 
-if ! [ -x "$(command -v xgo)" ]; then
-    go get github.com/karalabe/xgo
-fi
-
 DIR_BUILD="build/package"
 
-xgo \
-    -image=mysteriumnetwork/xgo:1.11 \
-    -ldflags="-w -s $(get_linker_ldflags)" \
-    -targets=android/* -out Mysterium -dest $DIR_BUILD \
- `pwd`/mobile/mysterium
+docker run --rm \
+    -v "$PWD"/$DIR_BUILD:/build \
+    -v "$GOPATH"/.xgo-cache:/deps-cache:ro \
+    -v "$GOPATH"/src:/ext-go/1/src:ro \
+    -e OUT=Mysterium \
+    -e FLAG_V=false \
+    -e FLAG_X=false \
+    -e FLAG_RACE=false \
+    -e FLAG_LDFLAGS="-w -s $(get_linker_ldflags)" \
+    -e FLAG_BUILDMODE=default \
+    -e TARGETS=android/. \
+    -e EXT_GOPATH=/ext-go/1 \
+    mysteriumnetwork/xgo:1.11 github.com/mysteriumnetwork/node/mobile/mysterium
```

Comment:
Here you can see that android build uses mysteriumnetwork/xgo:1.11 with xgo:1.11. I will fix it in separate PR as it will require to update this image to go1.13 and find gomobile commit which was still compatible with this android.

Reply:
Does this mean we can't use 1.13 features without breaking android build?